frontend/src/App.js

```diff
@@ -12,6 +13,10 @@ function App() {
           <Switch>
             <Route exact path="/" component={Homepage} />
             <Route path="/reviews/:restaurantId" component={Reviews} />
+            // TODO: The path for this endpoint should just be /restaurants, but since we share ports with the
+            // backend service which also has the endpoint /restaurants, we need to come up with a different name for
+            // now. Eventually, we should migrate this to /restaurants.
+            <Route path="/restaurantspage" component={Restaurants} />
```

Comment:
My understanding from two minutes of research is that the general convention for a URL route should use kebab case, so in this case "restaurants-page", in which we separate individual words with a hyphen

Reply:
makes sense to me!

Reply:
Fixed in [9b99bcd](https://github.com/NickPriv/FryRank/pull/17/commits/9b99bcd1c8795461a981ef323ec657bb2d09b4a1).